client/CS130_Team13/Assets/Scripts/Board/BaseObject.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NewBehaviourScript : MonoBehaviour
```

Comment:
nit: uncertain about unity: if the file is BaseObject, is it expected for the class to be called BaseObject, (and not class NewBehaviorScript)?